sisl/physics/tests/test_hamiltonian.py

```diff
@@ -615,6 +615,19 @@ def test_berry_phase_loop(self, setup):
         bz2 = BandStructure.param_circle(H, 20, 0.01, [0, 0, 1], [1/3] * 3, loop=True)
         assert np.allclose(elec.berry_phase(bz1), elec.berry_phase(bz2))
 
+    def test_berry_phase_zak(self):
```

Comment:
Great!

Reply:
How do I run this test?

Reply:
You install sisl locally, then open a new terminal (pytest doesn't work while running in the sisl directory).

Then do:
```bash
pytest --pyargs sisl -m hamiltonian
```
Which will only run the tests marked with `hamiltonian`. Alternatively, to test only one test I typically do:

```python
@pytest.mark.only
def test_berry_phase_zak(self):
    ...
```
Then you can do:
```bash
pytest --pyargs sisl -m only
```
to only run that test (or if you mark more as `only`).

Reply:
Thanks. Running the test I found a namespace issue and fixed it.